### PR TITLE
[VEN-2730]: Multichain governance on OP sepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves VEN-2730

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -7,18 +7,21 @@ export type SUPPORTED_NETWORKS =
...
 
 export type REMOTE_NETWORKS =
   | "sepolia"
   | "ethereum"
   | "opbnbtestnet"
   | "opbnbmainnet"
   | "arbitrumsepolia"
-  | "arbitrumone";
+  | "arbitrumone"
+  | "opsepolia";
```
Comment (GitGuru7) [inline src/types.ts:21]:
```suggestion
  | "opsepolia"
  | "opmainnet";
```

Comment (Debugger022) [reply src/types.ts:21]: I have not added the configurations for `opmainnet` and due to which checks will fail if I add `opmainnet` to `REMOTE_NETWORKS`.

**Review by chechu — COMMENTED** [review]
Sepolia commands lgtm

**Comment by Debugger022** [comment]
VIP: https://venus-testnet.on-fleek.app/#/governance/proposal/462?chainId=97
OP sepolia: https://sepolia-optimism.etherscan.io/tx/0x986914886b5cea4fe0aad632f234a870bf67d02ef31b06c661142d1a90c26d61
bsctestnet: https://testnet.bscscan.com/tx/0xa85fa268e33f3fb3dbedaf907be66ebf314474d9c0ce1474c9305292fd8d2e4d

**Comment by chechu** [comment]
To be refactored and merged after merging https://github.com/VenusProtocol/vips/pull/384

**Comment by chechu** [comment]
Already included at https://github.com/VenusProtocol/vips/pull/384

**Pull request merged**